test/common/router/config_impl_test.cc

```diff
@@ -2262,20 +2266,20 @@ name: foo
     }
     {
       Http::TestHeaderMapImpl headers =
-          genRedirectHeaders("redirect.lyft.com", "/foo", false, false);
-      EXPECT_EQ("http://new.lyft.com/foo",
+          genRedirectHeaders("redirect.lyft.com", "/host", false, false);
+      EXPECT_EQ("http://new.lyft.com/host",
                 config.route(headers, 0)->directResponseEntry()->newPath(headers));
     }
     {
       Http::TestHeaderMapImpl headers =
-          genRedirectHeaders("redirect.lyft.com", "/bar", true, false);
-      EXPECT_EQ("https://redirect.lyft.com/new_bar",
+          genRedirectHeaders("redirect.lyft.com", "/path", false, false);
```

Comment:
Looking at these more carefully I believe we've lost test coverage that https scheme is preserved in the non-redirect path. Let me know if I'm missing a test, otherwise please leave the tests which previously had true for ssl true?